src/common/variables.c

```diff
@@ -262,49 +338,39 @@ static char *get_base_value(dt_variables_params_t *params, char **variable)
       result = res;
     }
   }
-  else if(has_prefix(variable, "EXIF_APERTURE"))
+  else if(_has_prefix(variable, "EXIF_APERTURE"))
```

Comment:
`EXIF_APERTURE` -> `EXIF.APERTURE`

For compatibility we want to keep old namings.